core/common/src/main/java/alluxio/grpc/GrpcChannelKey.java

```diff
@@ -229,23 +234,26 @@ public GrpcChannelKey setKeepAliveTimeout(long keepAliveTimeout, TimeUnit timeUn
   }
 
   /**
-   *
-   * @param strategy the pooling strategy
+   * @param group the multiplexing group membership card
    * @return the modified {@link GrpcChannelKey}
    */
-  public GrpcChannelKey setPoolingStrategy(PoolingStrategy strategy) {
-    // TODO(feng): implement modularized pooling strategies
-    switch (strategy) {
+  public GrpcChannelKey setMultiplexGroup(MultiplexGroup group) {
+    mMultiplexGroup = group;
+
+    int groupSize = 1;
+    switch (mMultiplexGroup) {
       case DEFAULT:
-        mPoolKey = 0L;
+        groupSize = 1;
         break;
-      case DISABLED:
-        mPoolKey = RANDOM.nextLong();
+      case STREAMING:
+        groupSize = mConf.getInt(PropertyKey.USER_BLOCK_WORKER_MAX_STREAMING_CONNECTIONS);
```

Comment:
instead of doing a conf lookup every time, and since the conf is static, can we look up the group size once in the constructor and store that?